### DISCUSSION TITLE
Reorder asms tracks in BP

### DISCUSSION THREAD

**Discussion by selormsedofia** [jhomlala/betterplayer#685] [Q&A]
Hello,

How can the default asms tracks picked by BP from streams be reordered according to bit rates, or resolution?

Regards

**Comment by jhomlala** [comment]
There's no way to order tracks in BP in ASMS picker widget. I recommend you forking BP and adding sort option. You can do it easily here: https://github.com/jhomlala/betterplayer/blob/master/lib/src/controls/better_player_controls_state.dart#L276
